src/unreadCards/TopicUnreadCount.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+import React, { PureComponent } from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    width: 20,
+    height: 20,
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderRadius: 100,
+    padding: 3,
```

Comment:
Consider generalizing all these sizes, if they are to be reused. It looks like they might be.